### DISCUSSION TITLE
PCI Compliance

### DISCUSSION THREAD

**Discussion by g2ktcf** [zencart/zencart#4817] [Q&A] [answered]
In discussing features for ZC as it relates to our competition, PCI compliance was brought up.  The original announcement was here
https://www.zen-cart.com/showthread.php?216298-PCI-PA-DSS-Certification-Approved-for-Zen-Cart-v1-5-4

This was specifically or 1.5.4.  Where do we stand on compliance with 1.5.7d?  Is this a process that we need to revisit for 1.5.8?

**Comment by g2ktcf** [comment]
![image](https://user-images.githubusercontent.com/3268361/170286744-ddd9a0b0-6329-42c2-812c-602124cc232d.png)


Looked it up...and that certification has expired.  Should be push or renewal?  And how do addin/mods affect this?

**Comment by zcwilt** [comment, marked as answer]
Not going to happen.
PA-DSS  is a long and expensive process, would guess today circa $25-$40k
